hack/test-cmd.sh

```diff
@@ -74,6 +74,25 @@ function check-curl-proxy-code()
   echo "For address ${full_address}, got ${status} but wanted ${desired}"
   return 1
 }
+
+# TODO: remove this function when we do the retry inside the kubectl commands.
+function kubectl-with-retry()
+{
+  retry=true
+  count=0
+  ERROR_FILE="/tmp/kubectl-error"
```

Comment:
If multiple of these run in parallel, they will trample on each other.  You need to use a unique file name here.  @ixdy can advise on the best library to use.